shopfloor/actions/data.py

```diff
@@ -316,6 +317,15 @@ def _product_packaging(self, rec, field):
             multi=True,
         )
 
+    def _packaging_icon_data(self, rec, field):
+        icon_data = {"alt_text": rec.packaging_type_id.name}
+        if not rec.packaging_type_id.shopfloor_icon:
+            return icon_data
+        icon_data["url"] = "/web/image/product.packaging.type/{}/shopfloor_icon/".format(
```

Comment:
I still don't see the size here. If you don't use it, you have chances to load a bigger image.
What you do on the front is ok (forcing the width) but here we should try to load the smallest image possible.